scene/gui/tree.cpp

```diff
@@ -2923,8 +2923,7 @@ void Tree::item_selected(int p_column,TreeItem *p_item) {
 
 void Tree::item_deselected(int p_column,TreeItem *p_item) {
 
-	if (select_mode==SELECT_MULTI) {
-
+    if (select_mode==SELECT_MULTI || select_mode == SELECT_SINGLE) {
```

Comment:
Indentation issue, you should be using tabs like in the rest of the code, but your editor is likely enforcing 4 spaces. (same thing in the other commit). Could you rebase to fix them?